src/pages/salesProducts/SalesProducts.jsx

```diff
@@ -0,0 +1,133 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import useResize from "../../lib/hooks/useResize";
+
+import SalesProductCardList from "../../components/salesProducts/SalesProductCardList";
+import Title from "../../core/titles/Title";
+import fetchProduct from "../../lib/api/product";
+
+import "./sales-products.css";
+import SearchInput from "../../core/search/SearchInput";
+import Dropdown from "../../core/dropdown/Dropdown";
+import BtnSmall from "../../core/buttons/BtnSmall";
+import PagiNation from "../../components/salesProducts/PagiNation";
+
+const SalesProducts = () => {
+  const [salesProducts, setSalesProducts] = useState([]);
+  const [order, setOrder] = useState("recent");
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPage, setTotalPage] = useState(0);
```

Comment:
이 세가지의 상태들은 API 요청할때 Query라는 이름으로 항상 같이 다닐꺼에요. 그럴 경우에 각각 따로 쓰기보다 
하나의 객체로 선언해주어서 통째로 들고다니는게 편할때도 있습니다.

```suggestion
  const [productQuery,setProductQuery ] = useState(
                                                  { order: "recent", currentPage: 1, totalPage: 0 }
                                                  );
```